mmdet/models/losses/iou_loss.py

```diff
@@ -320,18 +335,18 @@ def __init__(self,
         self.loss_weight = loss_weight
 
     def forward(self,
-                pred,
-                target,
-                weight=None,
-                avg_factor=None,
-                reduction_override=None,
+                pred: Tensor,
+                target: Tensor,
+                weight: Optional[Tensor] = None,
+                avg_factor: Optional[int] = None,
+                reduction_override: Optional[str] = None,
                 **kwargs):
```

Comment:
```suggestion
                **kwargs) -> Tensor:
```

Not sure, but need have a check. Also, Docstrings need update. Such as 
```
Returns: 
    Tensor: xxx
```